Assets/Scripts/Level/LevelLoader.cs

```diff
@@ -17,6 +17,21 @@ private void Awake()
 
     private void onClick()
```

Comment:
Rename the method name to be more specific. Your method name should specify what it does.